src/patternfly/components/Banner/banner.scss

```diff
@@ -57,4 +66,45 @@
     z-index: var(--pf-c-banner--m-sticky--ZIndex);
     box-shadow: var(--pf-c-banner--m-sticky--BoxShadow);
   }
+
+  a {
+    color: var(--pf-c-banner--link--Color);
+    text-decoration: var(--pf-c-banner--link--TextDecoration);
+
+    &:hover:not(.pf-m-disabled) {
+      --pf-c-banner--link--Color: var(--pf-c-banner--link--hover--Color);
+
+      font-weight: var(--pf-c-banner--link--hover--FontWeight);
+    }
+
+    // stylelint-disable selector-no-qualifying-type
+    &.pf-m-disabled {
+      --pf-c-banner--link--Color: var(--pf-c-banner--link--disabled--Color);
+      --pf-c-banner--link--TextDecoration: var(--pf-c-banner--link--disabled--TextDecoration);
+
+      cursor: not-allowed;
+    }
+    // stylelint-enable selector-no-qualifying-type
+  }
+
+  .pf-m-inline {
```

Comment:
This needs to be scoped so it doesn't apply styles to some other component with `.pf-m-inline`